pkg/network/admitter/netsource.go

```diff
@@ -26,27 +26,43 @@ import (
 	k8sfield "k8s.io/apimachinery/pkg/util/validation/field"
 
 	v1 "kubevirt.io/api/core/v1"
+
```

Comment:
Is there a reason why this commit "net, admitter: Move additional network-source validations" can't be squashed with the former "Move network-source" commit?

Reply:
It may be possible, but this moves different validations that existence.
I thought this is a good approach as it helps follow what was moved exactly.

If I squash, I think it will be harder to review.

But as you are the reviewer, you could argue differently.
I do not mind squashing, I just mind making it reviewable.

Reply:
I don't necessarily think it should be squashed, but if they are not then the commit-msgs should differentiate the commits accordingly. could you please add a bit more info in these commit messages?

Reply:
In the previous `network-source` commit, the text mentions
> from the `vmi-create-admitter`move the single network source validation.

I think that is specific to what was moved there.

I added information in this commit to describe specifically what was moved, although I suspect these are all source-network checks that have been left.

Done